### PR TITLE
Update tests

### PR DESCRIPTION
**Pull request by zachkont**
image

### CODE REVIEW

File: tests/game.spec.ts [hunk 1]
```diff
@@ -0,0 +1,46 @@
+import Game from '../src/game';
+
+jest.mock('request-promise-native');
+
+// eslint-disable-next-line jest/no-disabled-tests
+describe('Game', () => {
+  let testGame: Game;
```
Comment (TimJentzsch) [inline tests/game.spec.ts:7]: Is the eslint disable still needed?

File: tests/user.spec.ts [hunk 2]
```diff
@@ -0,0 +1,25 @@
+import User, { UserRole } from '../src/user';
+import Channel from '../src/channel';
+import MockBot from './__mocks__/mockBot';
+
+let testUser: User;
+let testChannel: Channel;
+
+// eslint-disable-next-line jest/no-focused-tests
+describe('User', () => {
+  beforeAll(() => {
```
Comment (TimJentzsch) [inline tests/user.spec.ts:10]: Is the eslint disable still needed?

File: tests/channel.spec.ts [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+import MockBot from './__mocks__/mockBot';
+
+jest.mock('../src/managers/data_manager');
+
+const mockBot = new MockBot();
+let testChannel: Channel;
+
+// eslint-disable-next-line jest/no-disabled-tests
+describe.skip('Channel', () => {
+  beforeAll(() => {
```
Comment (TimJentzsch) [inline tests/channel.spec.ts:12]: Is this intentionally skipped?

**Review by TimJentzsch — CHANGES_REQUESTED** [review]

File: tests/channel.spec.ts (hunk 3 above)
Comment (zachkont) [reply tests/channel.spec.ts:12]: yeah, needs more work

**Review by TimJentzsch — APPROVED** [review]

**Pull request merged**